src/client/models/audioFilePath.ts

```diff
@@ -63,7 +65,9 @@ const audioFilePaths = [
   swallowing,
   throatClear,
   vomit,
-  wheezing
+  wheezing,
+  underwater
```

Comment:
Missing `,` character:

```suggestion
  underwater,
```